### DISCUSSION TITLE
ZKsync deploy successfully ---but not show verification successful...

### DISCUSSION THREAD

**Discussion by saif1163** [Cyfrin/foundry-full-course-cu#3122] [Q&A] [answered]
Hi, I m currently facing problem , my remix show Zksync deploy successful but show "confirmations": 0 and don't  show anything like verification  successful .and I compiled the code don't show any error but remix don't let me click on verify contract box , metamask is connected with zksync sepolia testnet , I clean browser cache and refresh remix many times  but still not show verification successful
![Screenshot (8)](https://github.com/user-attachments/assets/c0ed3ab6-a256-456f-a0ae-72c3df4ec4fd)

**Comment by H-code9** [comment, marked as answer]
Hi @saif1163,
1. navigate to the file explorer.
2. create a new folder named contracts.
3. copy your solidity file or your contract file and drop it in the contract folder.
4. compile your contract again, deploy and verify. Goodluck!!

**Reply by EngrPips** [reply]
Great Heads-up.

**Reply by saif1163** [reply]
thank very much its work.
